transmogrifier/sources/marc.py

```diff
@@ -181,6 +191,12 @@ def get_optional_fields(self, xml: Tag) -> dict:
         fields["contributors"] = contributor_values or None
 
         # dates
+        if fixed_length_data:
```

Comment:
I kept this simple for now but I'll add handling the 2nd date in the ticket I create